### ISSUE TITLE
I can't run my project moved from windows to mac please help.

### ISSUE THREAD

**Issue by furkann66** [flutter/flutter#88551]
Hello, I moved the project I wrote in Windows Flutter to a mac computer to build it to ios, but when I moved it, errors appeared and I can't solve it. I can't run it in the simulator, I keep getting an error, I would appreciate it if you could help.

The errors I'm getting now

<details> 
<summary>logs</summary> 
	
```console
Launching lib/main.dart on iPhone 12 Pro in debug mode...
Running Xcode build...
Xcode build done.                                            5.5s
Failed to build iOS app
Error output from Xcode build:
↳
    objc[16868]: Class AMSupportURLConnectionDelegate is implemented in both ?? (0x1f4898188) and ?? (0x1181b42b8). One of the two will be used. Which one is undefined.
    objc[16868]: Class AMSupportURLSession is implemented in both ?? (0x1f48981d8) and ?? (0x1181b4308). One of the two will be used. Which one is undefined.
    ** BUILD FAILED **


Xcode's output:
↳
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:22:43: error: expected member name or constructor call after type name
        private var preferredStatusBarStyle = UIStatusBarStyle
                                              ^
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:22:43: note: add arguments after the type to construct a value of the type
        private var preferredStatusBarStyle = UIStatusBarStyle
                                              ^
                                                              ()
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:22:43: note: use '.self' to reference the type object
        private var preferredStatusBarStyle = UIStatusBarStyle
                                              ^
                                                              .self
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:54:39: error: expected member name or constructor call after type name
            if preferredStatusBarStyle == UIStatusBarStyle {
                                          ^
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:54:39: note: add arguments after the type to construct a value of the type
            if preferredStatusBarStyle == UIStatusBarStyle {
                                          ^
                                                          ()
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:54:39: note: use '.self' to reference the type object
            if preferredStatusBarStyle == UIStatusBarStyle {
                                          ^
                                                          .self
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:55:75: error: cannot assign value of type 'Int' to type 'UIStatusBarStyle.Type'
                preferredStatusBarStyle = UIApplication.shared.statusBarStyle.rawValue
                                          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserWebViewController.swift:37:35: error: expected member name or constructor call after type name
        var preferredStatusBarStyle = UIStatusBarStyle
                                      ^
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserWebViewController.swift:37:35: note: add arguments after the type to construct a value of the type
        var preferredStatusBarStyle = UIStatusBarStyle
                                      ^
                                                      ()
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserWebViewController.swift:37:35: note: use '.self' to reference the type object
        var preferredStatusBarStyle = UIStatusBarStyle
                                      ^
                                                      .self
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:67:53: error: expected member name or constructor call after type name
            webViewController.preferredStatusBarStyle = UIStatusBarStyle
                                                        ^
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:67:53: note: add arguments after the type to construct a value of the type
            webViewController.preferredStatusBarStyle = UIStatusBarStyle
                                                        ^
                                                                        ()
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:67:53: note: use '.self' to reference the type object
            webViewController.preferredStatusBarStyle = UIStatusBarStyle
                                                        ^
                                                                        .self
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserWebViewController.swift:37:35: error: expected member name or constructor call after type name
        var preferredStatusBarStyle = UIStatusBarStyle
                                      ^
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserWebViewController.swift:37:35: note: add arguments after the type to construct a value of the type
        var preferredStatusBarStyle = UIStatusBarStyle
                                      ^
                                                      ()
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserWebViewController.swift:37:35: note: use '.self' to reference the type object
        var preferredStatusBarStyle = UIStatusBarStyle
                                      ^
                                                      .self
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserWebViewController.swift:37:9: error: overriding property must be as accessible as its enclosing type
        var preferredStatusBarStyle = UIStatusBarStyle
            ^
        public 
    UIKit.UIViewController:174:14: note: overridden declaration is here
        open var preferredStatusBarStyle: UIStatusBarStyle { get }
                 ^
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserWebViewController.swift:37:9: error: property 'preferredStatusBarStyle' with type 'UIStatusBarStyle.Type' cannot override a property with type 'UIStatusBarStyle'
        var preferredStatusBarStyle = UIStatusBarStyle
            ^
    UIKit.UIViewController:174:14: note: attempt to override property here
        open var preferredStatusBarStyle: UIStatusBarStyle { get }
                 ^
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserWebViewController.swift:556:39: error: expected member name or constructor call after type name
            if preferredStatusBarStyle != UIStatusBarStyle {
                                          ^
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserWebViewController.swift:556:39: note: add arguments after the type to construct a value of the type
            if preferredStatusBarStyle != UIStatusBarStyle {
                                          ^
                                                          ()
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserWebViewController.swift:556:39: note: use '.self' to reference the type object
            if preferredStatusBarStyle != UIStatusBarStyle {
                                          ^
                                                          .self
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserWebViewController.swift:557:78: error: cannot convert value of type 'UIStatusBarStyle' to expected argument type 'Int'
                UIApplication.shared.statusBarStyle = UIStatusBarStyle(rawValue: preferredStatusBarStyle)!
                                                                                 ^
                                                                                          .rawValue
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppWebView/InAppWebView.swift:517:31: warning: 'mediaPlaybackRequiresUserAction' was deprecated in iOS 9.0
                    configuration.mediaPlaybackRequiresUserAction = options.mediaPlaybackRequiresUserGesture
                                  ^
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppWebView/InAppWebView.swift:928:31: warning: 'mediaPlaybackRequiresUserAction' was deprecated in iOS 9.0
                    configuration.mediaPlaybackRequiresUserAction = newOptions.mediaPlaybackRequiresUserGesture
                                  ^
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppWebView/InAppWebViewOptions.swift:102:81: warning: 'mediaPlaybackRequiresUserAction' was deprecated in iOS 9.0
                    realOptions["mediaPlaybackRequiresUserGesture"] = configuration.mediaPlaybackRequiresUserAction
                                                                                    ^
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/Types/URLCredential.swift:14:25: warning: comparing non-optional value of type '[Any]' to 'nil' always returns true
            if certificates != nil {
               ~~~~~~~~~~~~ ^  ~~~
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:22:43: error: expected member name or constructor call after type name
        private var preferredStatusBarStyle = UIStatusBarStyle
                                              ^
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:22:43: note: add arguments after the type to construct a value of the type
        private var preferredStatusBarStyle = UIStatusBarStyle
                                              ^
                                                              ()
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:22:43: note: use '.self' to reference the type object
        private var preferredStatusBarStyle = UIStatusBarStyle
                                              ^
                                                              .self
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/Util.swift:216:21: warning: immutable value 'j' was never used; consider replacing with '_' or removing it
                    for j in ipv6.count...8 {
                        ^
                        _
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:22:43: error: expected member name or constructor call after type name
        private var preferredStatusBarStyle = UIStatusBarStyle
                                              ^
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:22:43: note: add arguments after the type to construct a value of the type
        private var preferredStatusBarStyle = UIStatusBarStyle
                                              ^
                                                              ()
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:22:43: note: use '.self' to reference the type object
        private var preferredStatusBarStyle = UIStatusBarStyle
                                              ^
                                                              .self
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:54:39: error: expected member name or constructor call after type name
            if preferredStatusBarStyle == UIStatusBarStyle {
                                          ^
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:54:39: note: add arguments after the type to construct a value of the type
            if preferredStatusBarStyle == UIStatusBarStyle {
                                          ^
                                                          ()
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:54:39: note: use '.self' to reference the type object
            if preferredStatusBarStyle == UIStatusBarStyle {
                                          ^
                                                          .self
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:55:75: error: cannot assign value of type 'Int' to type 'UIStatusBarStyle.Type'
                preferredStatusBarStyle = UIApplication.shared.statusBarStyle.rawValue
                                          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserWebViewController.swift:37:35: error: expected member name or constructor call after type name
        var preferredStatusBarStyle = UIStatusBarStyle
                                      ^
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserWebViewController.swift:37:35: note: add arguments after the type to construct a value of the type
        var preferredStatusBarStyle = UIStatusBarStyle
                                      ^
                                                      ()
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserWebViewController.swift:37:35: note: use '.self' to reference the type object
        var preferredStatusBarStyle = UIStatusBarStyle
                                      ^
                                                      .self
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:67:53: error: expected member name or constructor call after type name
            webViewController.preferredStatusBarStyle = UIStatusBarStyle
                                                        ^
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:67:53: note: add arguments after the type to construct a value of the type
            webViewController.preferredStatusBarStyle = UIStatusBarStyle
                                                        ^
                                                                        ()
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:67:53: note: use '.self' to reference the type object
            webViewController.preferredStatusBarStyle = UIStatusBarStyle
                                                        ^
                                                                        .self
    Command CompileSwift failed with a nonzero exit code
    Command CompileSwift failed with a nonzero exit code
    Command CompileSwift failed with a nonzero exit code
    Command CompileSwift failed with a nonzero exit code
    Command CompileSwift failed with a nonzero exit code
    Command CompileSwift failed with a nonzero exit code
    Command CompileSwift failed with a nonzero exit code
    Command CompileSwift failed with a nonzero exit code
    Command CompileSwift failed with a nonzero exit code
    Command CompileSwift failed with a nonzero exit code
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/Types/URLCredential.swift:14:25: warning: comparing non-optional value of type '[Any]' to 'nil' always returns true
            if certificates != nil {
               ~~~~~~~~~~~~ ^  ~~~
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:22:43: error: expected member name or constructor call after type name
        private var preferredStatusBarStyle = UIStatusBarStyle
                                              ^
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:22:43: note: add arguments after the type to construct a value of the type
        private var preferredStatusBarStyle = UIStatusBarStyle
                                              ^
                                                              ()
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:22:43: note: use '.self' to reference the type object
        private var preferredStatusBarStyle = UIStatusBarStyle
                                              ^
                                                              .self
    /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/Util.swift:216:21: warning: immutable value 'j' was never used; consider replacing with '_' or removing it
                    for j in ipv6.count...8 {
                        ^
                        _
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Analyzing workspace
    note: Constructing build description
    note: Build preparation complete

Could not build the application for the simulator.
Error launching application on iPhone 12 Pro.
```
	
</details>

**Comment by furkann66** [comment]
![Screen Shot 2021-08-20 at 00 58 24](https://user-images.githubusercontent.com/89225717/130149944-f1d41b3a-7dbd-479a-86b3-64e6dae37d24.png)

**Comment by furkann66** [comment]
> #


> Hello, I moved the project I wrote in Windows Flutter to a mac computer to build it to ios, but when I moved it, errors appeared and I can't solve it. I can't run it in the simulator, I keep getting an error, I would appreciate it if you could help.
> 
> The errors I'm getting now
> 
> Launching lib/main.dart on iPhone 12 Pro in debug mode...
> Running Xcode build...
> Xcode build done. 5.5s
> Failed to build iOS app
> Error output from Xcode build:
> ↳
> objc[16868]: Class AMSupportURLConnectionDelegate is implemented in both ?? (0x1f4898188) and ?? (0x1181b42b8). One of the two will be used. Which one is undefined.
> objc[16868]: Class AMSupportURLSession is implemented in both ?? (0x1f48981d8) and ?? (0x1181b4308). One of the two will be used. Which one is undefined.
> ** BUILD FAILED **
> 
> Xcode's output:
> ↳
> /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:22:43: error: expected member name or constructor call after type name
> private var preferredStatusBarStyle = UIStatusBarStyle
> ^
> /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:22:43: note: add arguments after the type to construct a value of the type
> private var preferredStatusBarStyle = UIStatusBarStyle
> ^
> ()
> /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:22:43: note: use '.self' to reference the type object
> private var preferredStatusBarStyle = UIStatusBarStyle
> ^
> .self
> /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:54:39: error: expected member name or constructor call after type name
> if preferredStatusBarStyle == UIStatusBarStyle {
> ^
> /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:54:39: note: add arguments after the type to construct a value of the type
> if preferredStatusBarStyle == UIStatusBarStyle {
> ^
> ()
> /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:54:39: note: use '.self' to reference the type object
> if preferredStatusBarStyle == UIStatusBarStyle {
> ^
> .self
> /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:55:75: error: cannot assign value of type 'Int' to type 'UIStatusBarStyle.Type'
> preferredStatusBarStyle = UIApplication.shared.statusBarStyle.rawValue
> ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~
> /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserWebViewController.swift:37:35: error: expected member name or constructor call after type name
> var preferredStatusBarStyle = UIStatusBarStyle
> ^
> /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserWebViewController.swift:37:35: note: add arguments after the type to construct a value of the type
> var preferredStatusBarStyle = UIStatusBarStyle
> ^
> ()
> /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserWebViewController.swift:37:35: note: use '.self' to reference the type object
> var preferredStatusBarStyle = UIStatusBarStyle
> ^
> .self
> /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:67:53: error: expected member name or constructor call after type name
> webViewController.preferredStatusBarStyle = UIStatusBarStyle
> ^
> /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:67:53: note: add arguments after the type to construct a value of the type
> webViewController.preferredStatusBarStyle = UIStatusBarStyle
> ^
> ()
> /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:67:53: note: use '.self' to reference the type object
> webViewController.preferredStatusBarStyle = UIStatusBarStyle
> ^
> .self
> /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserWebViewController.swift:37:35: error: expected member name or constructor call after type name
> var preferredStatusBarStyle = UIStatusBarStyle
> ^
> /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserWebViewController.swift:37:35: note: add arguments after the type to construct a value of the type
> var preferredStatusBarStyle = UIStatusBarStyle
> ^
> ()
> /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserWebViewController.swift:37:35: note: use '.self' to reference the type object
> var preferredStatusBarStyle = UIStatusBarStyle
> ^
> .self
> /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserWebViewController.swift:37:9: error: overriding property must be as accessible as its enclosing type
> var preferredStatusBarStyle = UIStatusBarStyle
> ^
> public
> UIKit.UIViewController:174:14: note: overridden declaration is here
> open var preferredStatusBarStyle: UIStatusBarStyle { get }
> ^
> /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserWebViewController.swift:37:9: error: property 'preferredStatusBarStyle' with type 'UIStatusBarStyle.Type' cannot override a property with type 'UIStatusBarStyle'
> var preferredStatusBarStyle = UIStatusBarStyle
> ^
> UIKit.UIViewController:174:14: note: attempt to override property here
> open var preferredStatusBarStyle: UIStatusBarStyle { get }
> ^
> /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserWebViewController.swift:556:39: error: expected member name or constructor call after type name
> if preferredStatusBarStyle != UIStatusBarStyle {
> ^
> /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserWebViewController.swift:556:39: note: add arguments after the type to construct a value of the type
> if preferredStatusBarStyle != UIStatusBarStyle {
> ^
> ()
> /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserWebViewController.swift:556:39: note: use '.self' to reference the type object
> if preferredStatusBarStyle != UIStatusBarStyle {
> ^
> .self
> /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserWebViewController.swift:557:78: error: cannot convert value of type 'UIStatusBarStyle' to expected argument type 'Int'
> UIApplication.shared.statusBarStyle = UIStatusBarStyle(rawValue: preferredStatusBarStyle)!
> ^
> .rawValue
> /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppWebView/InAppWebView.swift:517:31: warning: 'mediaPlaybackRequiresUserAction' was deprecated in iOS 9.0
> configuration.mediaPlaybackRequiresUserAction = options.mediaPlaybackRequiresUserGesture
> ^
> /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppWebView/InAppWebView.swift:928:31: warning: 'mediaPlaybackRequiresUserAction' was deprecated in iOS 9.0
> configuration.mediaPlaybackRequiresUserAction = newOptions.mediaPlaybackRequiresUserGesture
> ^
> /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppWebView/InAppWebViewOptions.swift:102:81: warning: 'mediaPlaybackRequiresUserAction' was deprecated in iOS 9.0
> realOptions["mediaPlaybackRequiresUserGesture"] = configuration.mediaPlaybackRequiresUserAction
> ^
> /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/Types/URLCredential.swift:14:25: warning: comparing non-optional value of type '[Any]' to 'nil' always returns true
> if certificates != nil {
> ~~~~~~~~~~~~ ^ ~~~
> /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:22:43: error: expected member name or constructor call after type name
> private var preferredStatusBarStyle = UIStatusBarStyle
> ^
> /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:22:43: note: add arguments after the type to construct a value of the type
> private var preferredStatusBarStyle = UIStatusBarStyle
> ^
> ()
> /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:22:43: note: use '.self' to reference the type object
> private var preferredStatusBarStyle = UIStatusBarStyle
> ^
> .self
> /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/Util.swift:216:21: warning: immutable value 'j' was never used; consider replacing with '_' or removing it for j in ipv6.count...8 { ^ _ /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:22:43: error: expected member name or constructor call after type name private var preferredStatusBarStyle = UIStatusBarStyle ^ /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:22:43: note: add arguments after the type to construct a value of the type private var preferredStatusBarStyle = UIStatusBarStyle ^ () /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:22:43: note: use '.self' to reference the type object private var preferredStatusBarStyle = UIStatusBarStyle ^ .self /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:54:39: error: expected member name or constructor call after type name if preferredStatusBarStyle == UIStatusBarStyle { ^ /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:54:39: note: add arguments after the type to construct a value of the type if preferredStatusBarStyle == UIStatusBarStyle { ^ () /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:54:39: note: use '.self' to reference the type object if preferredStatusBarStyle == UIStatusBarStyle { ^ .self /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:55:75: error: cannot assign value of type 'Int' to type 'UIStatusBarStyle.Type' preferredStatusBarStyle = UIApplication.shared.statusBarStyle.rawValue ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~ /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserWebViewController.swift:37:35: error: expected member name or constructor call after type name var preferredStatusBarStyle = UIStatusBarStyle ^ /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserWebViewController.swift:37:35: note: add arguments after the type to construct a value of the type var preferredStatusBarStyle = UIStatusBarStyle ^ () /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserWebViewController.swift:37:35: note: use '.self' to reference the type object var preferredStatusBarStyle = UIStatusBarStyle ^ .self /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:67:53: error: expected member name or constructor call after type name webViewController.preferredStatusBarStyle = UIStatusBarStyle ^ /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:67:53: note: add arguments after the type to construct a value of the type webViewController.preferredStatusBarStyle = UIStatusBarStyle ^ () /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:67:53: note: use '.self' to reference the type object webViewController.preferredStatusBarStyle = UIStatusBarStyle ^ .self Command CompileSwift failed with a nonzero exit code Command CompileSwift failed with a nonzero exit code Command CompileSwift failed with a nonzero exit code Command CompileSwift failed with a nonzero exit code Command CompileSwift failed with a nonzero exit code Command CompileSwift failed with a nonzero exit code Command CompileSwift failed with a nonzero exit code Command CompileSwift failed with a nonzero exit code Command CompileSwift failed with a nonzero exit code Command CompileSwift failed with a nonzero exit code /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/Types/URLCredential.swift:14:25: warning: comparing non-optional value of type '[Any]' to 'nil' always returns true if certificates != nil { ~~~~~~~~~~~~ ^ ~~~ /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:22:43: error: expected member name or constructor call after type name private var preferredStatusBarStyle = UIStatusBarStyle ^ /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:22:43: note: add arguments after the type to construct a value of the type private var preferredStatusBarStyle = UIStatusBarStyle ^ () /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/InAppBrowser/InAppBrowserManager.swift:22:43: note: use '.self' to reference the type object private var preferredStatusBarStyle = UIStatusBarStyle ^ .self /Users/allrmac/FlutterDev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.2/ios/Classes/Util.swift:216:21: warning: immutable value 'j' was never used; consider replacing with '_' or removing it
> for j in ipv6.count...8 {
> ^
> _
> note: Using new build system
> note: Building targets in parallel
> note: Planning build
> note: Analyzing workspace
> note: Constructing build description
> note: Build preparation complete
> 
> Could not build the application for the simulator.
> Error launching application on iPhone 12 Pro.

Deleting inappwebview from pubspec.yaml gave these errors

Running "flutter pub get" in ftl-master...
Launching lib/main.dart on iPhone 12 Pro in debug mode...
Running pod install...
Running Xcode build...
Xcode build done.                                            8.3s
Failed to build iOS app
Error output from Xcode build:
↳
    objc[19147]: Class AMSupportURLConnectionDelegate is implemented in both ?? (0x1f4898188) and ?? (0x11b6d82b8). One of the two will be used. Which one is undefined.
    objc[19147]: Class AMSupportURLSession is implemented in both ?? (0x1f48981d8) and ?? (0x11b6d8308). One of the two will be used. Which one is undefined.
    ** BUILD FAILED **


Xcode's output:
↳
    /Users/allrmac/Downloads/ftl-master/ios/Runner/Runner-Bridging-Header.h:1:9: note: in file included from /Users/allrmac/Downloads/ftl-master/ios/Runner/Runner-Bridging-Header.h:1:
    #import "GeneratedPluginRegistrant.h"
            ^
    /Users/allrmac/Downloads/ftl-master/ios/Runner/GeneratedPluginRegistrant.h:10:9: error: 'Flutter/Flutter.h' file not found
    #import <Flutter/Flutter.h>
            ^
    1 error generated.
    <unknown>:0: error: failed to emit precompiled header '/Users/allrmac/Library/Developer/Xcode/DerivedData/Runner-btymkttkogownigjwvjmshzxjbig/Build/Intermediates.noindex/PrecompiledHeaders/Runner-Bridging-Header-swift_3364FZ5DJFUZK-clang_23A1ELK8R7557.pch' for bridging header '/Users/allrmac/Downloads/ftl-master/ios/Runner/Runner-Bridging-Header.h'
    /Users/allrmac/Downloads/ftl-master/ios/Runner/Runner-Bridging-Header.h:1:9: note: in file included from /Users/allrmac/Downloads/ftl-master/ios/Runner/Runner-Bridging-Header.h:1:
    #import "GeneratedPluginRegistrant.h"
            ^
    2 errors generated.
    <unknown>:0: error: generate-pch command failed with exit code 1 (use -v to see invocation)
    Command PrecompileSwiftBridgingHeader failed with a nonzero exit code
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Analyzing workspace
    note: Constructing build description
    note: Build preparation complete
    warning: Stale file '/Users/allrmac/Downloads/ftl-master/build/ios/Debug-iphonesimulator/Runner.app/Frameworks/OrderedSet.framework' is located outside of the allowed root paths.

    warning: Stale file '/Users/allrmac/Downloads/ftl-master/build/ios/Debug-iphonesimulator/Runner.app/Frameworks/flutter_inappwebview.framework' is located outside of the allowed root paths.

    note: Removed stale file '/Users/allrmac/Library/Developer/Xcode/DerivedData/Runner-btymkttkogownigjwvjmshzxjbig/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/PromisesObjC.build/Script-DEFE21BBE434421B788DC737E8A22EC8.sh'

    note: Removed stale file '/Users/allrmac/Library/Developer/Xcode/DerivedData/Runner-btymkttkogownigjwvjmshzxjbig/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/GoogleUserMessagingPlatform.build/InputFileList-21B06E9CBA3A4721B8D28761CE8C10F1-GoogleUserMessagingPlatform-xcframeworks-input-files-b527e04bb1adb3d15b52253c451f7fc4-resolved.xcfilelist'

    note: Removed stale file '/Users/allrmac/Library/Developer/Xcode/DerivedData/Runner-btymkttkogownigjwvjmshzxjbig/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/GoogleAppMeasurement.build/InputFileList-BFB18894A06B6D0BF8EA6201F8D5BDB7-GoogleAppMeasurement-xcframeworks-input-files-827eae77ecde48e4964763019863b065-resolved.xcfilelist'

    note: Removed stale file '/Users/allrmac/Library/Developer/Xcode/DerivedData/Runner-btymkttkogownigjwvjmshzxjbig/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/GoogleAppMeasurement.build/OutputFileList-BFB18894A06B6D0BF8EA6201F8D5BDB7-GoogleAppMeasurement-xcframeworks-output-files-23feb1f4826fdce139431596bad33e0e-resolved.xcfilelist'

    note: Removed stale file '/Users/allrmac/Library/Developer/Xcode/DerivedData/Runner-btymkttkogownigjwvjmshzxjbig/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/GoogleUserMessagingPlatform.build/OutputFileList-21B06E9CBA3A4721B8D28761CE8C10F1-GoogleUserMessagingPlatform-xcframeworks-output-files-d1ff448682209046843bdcb14d657e40-resolved.xcfilelist'

    note: Removed stale file '/Users/allrmac/Library/Developer/Xcode/DerivedData/Runner-btymkttkogownigjwvjmshzxjbig/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/GoogleAppMeasurement.build/Script-BFB18894A06B6D0BF8EA6201F8D5BDB7.sh'

    note: Removed stale file '/Users/allrmac/Library/Developer/Xcode/DerivedData/Runner-btymkttkogownigjwvjmshzxjbig/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/GoogleUserMessagingPlatform.build/Script-21B06E9CBA3A4721B8D28761CE8C10F1.sh'


Could not build the application for the simulator.
Error launching application on iPhone 12 Pro.

**Comment by rwintrich** [comment]
Chip m1?

**Comment by TahaTesser** [comment]
Hi @furkann66 
Your windows pub cache has different version pub packages hence the issue. 

Upgrade your pub cache macOS too with `flutter pub upgrade`, make sure your packages are up to date in your` pubspec.yaml`. and try `flutter pub get` and `flutter run -v`

**Comment by darshankawar** [comment]
Without additional information, we are unfortunately not sure how to resolve this issue. 
We are therefore reluctantly going to close this bug for now. 
Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!  
Thanks for your contribution.

Could everyone who still has this problem please file a new issue with the exact description of what happens, logs, and the output of `flutter doctor -v`.
All system setups can be slightly different, so it's always better to open new issues and reference related issues.

**Comment by furkann66** [comment]
Thanks for your help, I solved my problem by moving the files one by one.
